exercises/list-ops/canonical-data.json

```diff
@@ -184,18 +187,75 @@
         {
           "uuid": "d2cf5644-aee1-4dfc-9b88-06896676fe27",
           "description": "direction dependent function applied to non-empty list",
+          "comments": [
+            "Expects integer division (expects / to discard fractions).       "
+          ],
           "property": "foldl",
           "input": {
             "list": [2, 5],
             "initial": 5,
             "function": "(x, y) -> x / y"
```

Comment:
note for posterity: The possible ambiguous interpretation for this function were:

* acc, el -> acc / el
    * acc: 5, el: 2. new acc 0
    * acc: 0, el: 5. new acc 0
* acc, el -> el / acc
    * acc: 5, el: 2. new acc 0
    * acc: 0, el: 5. division by zero.

So clearly the **former** was intended. Note that this is different from the other case, where the **latter** was intended. The fact that these two same strings require two different interpretations is what's confusing and is the motivation for this PR.